packages/codemirror-extension/src/index.ts

```diff
@@ -135,6 +144,15 @@ function activateEditorServices(app: JupyterFrontEnd): IEditorServices {
   return editorServices;
 }
 
+/**
+ * Set up the CodeMirror singleton.
+ */
+function activateCodeMirror(app: JupyterFrontEnd): ICodeMirror {
+  return {
+    codemirrorSingleton: () => CodeMirror
```

Comment:
How about for consistency with the CodeMirror documention and simplicity, we just call this:
```suggestion
    get CodeMirror() { return CodeMirror }
```
That way it is the same name as in the codemirror docs, and it is an attribute rather than a function. Thoughts?

Reply:
I think you used a function for immutability. I suppose if we use `get` , we need to use an anonymous class:

```js
return new class { get CodeMirror() {return CodeMirror} }
```
We could also get immutability by freezing an object:

```js
let value = {CodeMirror};
Object.freeze(value);
return value
```